packages/api-explorer/e2e/diffScene.spec.ts

```diff
@@ -265,4 +270,70 @@ describe('Diff Scene', () => {
     expect(diff40to31Page1Methods).toHaveLength(15)
     expect(diff40to31Page1Methods).toContain('delete_board_item')
   })
+
+  it('updates when a comparison option is toggled', async () => {
```

Comment:
Can remove this, but I thought it would be nice to have an integration test ensuring the DiffScene operates properly other than the unit tests which I couldn't make comprehensive due to the spec file bug.